pkg/playback/core_playback.go

```diff
@@ -0,0 +1,210 @@
+package playback
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+func check(err error) {
```

Comment:
This should be removed since you only use it in the tests. Your test version should probably be something like
```go
func check(t *testing.T, err error) {
  t.Fatal(err)
}
```